src/components/ProductTours/BrowseAndRequestTour.jsx

```diff
@@ -12,19 +12,23 @@ import {
   BROWSE_AND_REQUEST_TOUR_COOKIE_NAME,
   TOUR_TARGETS,
 } from './constants';
+import { SubsidyRequestConfigurationContext } from '../subsidy-request-configuration';
 
 const cookies = new Cookies();
 
 const BrowseAndRequestTour = ({ enterpriseSlug, enableBrowseAndRequest }) => {
+  const { subsidyRequestConfiguration } = useContext(SubsidyRequestConfigurationContext);
+  const isEligibleForFeature = !!subsidyRequestConfiguration?.subsidyType;
```

Comment:
Should this also be adhering to `subsidyRequestConfiguration?.subsidyRequestsEnabled`?

Reply:
nope, subsidyRequestsEnabled defaults to false for all customers at the time of creation of the configuration. As long as the enterprise is eligible, the tour should be shown.